### PR TITLE
Opt Out API: Implement common opt out store interface

### PR DESCRIPTION
**Pull request by g1039**

### CODE REVIEW

**Comment by g1039** [comment]
@hodgestar Ready for review

**Comment by hodgestar** [comment]
Looks good.

Should we add a method on the interface for retrieving the number of opt outs? I.e. the count?

**Comment by g1039** [comment]
@hodgestar ready for review

File: store/interface.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+        def put(address_type, address):
+            """ Store a record of an opt out for an address. """
+            pass
+
+        def delete(address_type, address):
+            """ Remove an opt out for an address. """
+            pass
+
+        def count(results):
```
Comment (hodgestar) [inline store/interface.py]: Why does `count` take a `results` argument?

**Comment by hodgestar** [comment]
@TrevorOctober The `store` package needs to be inside the `opt_outs_http_api` folder.

**Comment by g1039** [comment]
@hodgestar Ready for re-review

File: opt_out_http_api/store/interface.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+        def put(address_type, address):
+            """ Store a record of an opt out for an address. """
+            pass
+
+        def delete(address_type, address):
+            """ Remove an opt out for an address. """
+            pass
+
+        def count(opt_out_count):
```
Comment (hodgestar) [inline opt_out_http_api/store/interface.py]: Does `count` need to take a parameter?

**Comment by hodgestar** [comment]
:+1:

**Pull request merged**